cluster-monitoring/readme.adoc

```diff
@@ -34,58 +34,69 @@ Now, Dashboard is accessible at http://localhost:8001/api/v1/namespaces/kube-sys
+  certificate-controller-token-p131b       kubernetes.io/service-account-token   3         3h
+  daemon-set-controller-token-r4mmp        kubernetes.io/service-account-token   3         3h
+  default-token-7vh0x                      kubernetes.io/service-account-token   3         3h
+  deployment-controller-token-jlzkj        kubernetes.io/service-account-token   3         3h
+  disruption-controller-token-qrx2v        kubernetes.io/service-account-token   3         3h
+  dns-controller-token-v49b6               kubernetes.io/service-account-token   3         3h
+  endpoint-controller-token-hgkbm          kubernetes.io/service-account-token   3         3h
+  generic-garbage-collector-token-34fvc    kubernetes.io/service-account-token   3         3h
+  horizontal-pod-autoscaler-token-lhbkf    kubernetes.io/service-account-token   3         3h
+  job-controller-token-c2s8j               kubernetes.io/service-account-token   3         3h
+  kube-dns-autoscaler-token-s3svx          kubernetes.io/service-account-token   3         3h
+  kube-dns-token-92xzb                     kubernetes.io/service-account-token   3         3h
+  kube-proxy-token-0ww14                   kubernetes.io/service-account-token   3         3h
+  kubernetes-dashboard-certs               Opaque                                2         9m
+  kubernetes-dashboard-key-holder          Opaque                                2         9m
+  kubernetes-dashboard-token-vt0fd         kubernetes.io/service-account-token   3         10m
+  namespace-controller-token-423gh         kubernetes.io/service-account-token   3         3h
+  node-controller-token-r6lsr              kubernetes.io/service-account-token   3         3h
+  persistent-volume-binder-token-xv30g     kubernetes.io/service-account-token   3         3h
+  pod-garbage-collector-token-fwmv4        kubernetes.io/service-account-token   3         3h
+  replicaset-controller-token-0cg8r        kubernetes.io/service-account-token   3         3h
+  replication-controller-token-3fwxd       kubernetes.io/service-account-token   3         3h
+  resourcequota-controller-token-6rl9f     kubernetes.io/service-account-token   3         3h
+  route-controller-token-9brzb             kubernetes.io/service-account-token   3         3h
+  service-account-controller-token-bqlsk   kubernetes.io/service-account-token   3         3h
+  service-controller-token-1qlg6           kubernetes.io/service-account-token   3         3h
+  statefulset-controller-token-kmgzg       kubernetes.io/service-account-token   3         3h
+  ttl-controller-token-vbnhf               kubernetes.io/service-account-token   3         3h
+
+We can login using the secret with type 'kubernetes.io/service-account-token'. In our case, we'll use the token from secret `k8s-namespace-sa-token-423gh` to login. Use the following command to get the token for this secret:
+
+    kubectl -n kube-system describe secret k8s-dashboad-sa-token-423gh
```

Comment:
k8s-dashboad-sa-token-423gh is missing from the output list. Can you please update?